ibm_catalog.json

```diff
@@ -0,0 +1,90 @@
+{
+    "products": [
+        {
+            "name": "deploy-arch-ibm-cos-config",
+            "label": "COS bucket configuration (sample DA)",
```

Comment:
Can we use a more use case friendly name since that is what we suggest when others create DAs? Maybe "Object Storage regional bucket replication"

Reply:
Not all variations of the offering will do replication, but they will all configure COS buckets in some way. Maybe I go with just `Object Storage bucket configuration` ?

Reply:
Sure. If we can abstract even more from our product name and align with the customer use case, that is what the guidance tells us to do. For example, if the variations were all opinionated ways to do storage configuration in a secure way we could use Secure storage something or other :) But, I get I might be pushing it for a sample repo.